src/main/scala/io/iohk/ethereum/nodebuilder/NodeBuilder.scala

```diff
@@ -74,7 +74,7 @@ trait AsyncConfigBuilder {
 }
 
 trait ActorSystemBuilder {
-  implicit lazy val system: ActorSystem = ActorSystem("mantis_system")
+  implicit lazy val system: ActorSystem = ActorSystem("mantis_system", classLoader = Some(getClass.getClassLoader))
```

Comment:
I think manual management of class loader for production code, to solve an issue with tests that fail when a run a couple of times might not be a good idea.

Reply:
Right, I hate this one too, but the `Fixture` used in tests also brings on the Mantis Node cake, which isn't good either.
I have two ideas around that: 
1) We could get rid of this by implementing a `Fixture` that will not trigger a whole `NodeBuilder`
    This can be serious refactoring for the system that is built all around that pattern
2) We can lean to a particular `ClassLoader` to prevent `ActorSystem` from voluntarily selecting one. 
    I'm thankful to failed tests and also suppose that `it-will-not-fail-in-production` is a kind of survivorship bias. The production environment is clear and isolated, of course, but that particular bug may arise in another circumstances.
    
    Personally, I'd never rely on environment and will apply maximum effort to guarantee a robust behavior in any.

Reply:
You can also remove "classloader" parameter in "production" part of code and overwrite it in tests only (and maybe even extract the override to a trait)